src/dbnode/client/session.go

```diff
@@ -68,13 +68,11 @@ import (
 )
 
 const (
-	clusterConnectWaitInterval           = 10 * time.Millisecond
-	blocksMetadataChannelInitialCapacity = 4096
-	gaugeReportInterval                  = 500 * time.Millisecond
-	blockMetadataChBufSize               = 4096
-	shardResultCapacity                  = 4096
-	hostNotAvailableMinSleepInterval     = 1 * time.Millisecond
-	hostNotAvailableMaxSleepInterval     = 100 * time.Millisecond
+	clusterConnectWaitInterval       = 10 * time.Millisecond
+	gaugeReportInterval              = 500 * time.Millisecond
+	blockMetadataChBufSize           = 65536
```

Comment:
That's quite a substantial increase, is this intentional?

Reply:
Yes, this was unfortunately a significant bottleneck since IO work is required to pull off the queue here and block the producer frequently. Also the individual channel element struct is rather small so this doesn't add much overall space / usage.